### ISSUE TITLE
cibuildwheel v2.16.2 fails with PyPy 39 on macosx.

### ISSUE THREAD

**Issue by charris** [pypa/cibuildwheel#1642]
### Description

See https://github.com/numpy/numpy/actions/runs/6508239109/job/17677221023?pr=24920.  Version v2.16.1 works.
```
  + cp -r /etc/ssl/cert.pem /etc/ssl/certs /etc/ssl/cert.pem /etc/ssl/certs /var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/openblas/lib/cmake /var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/openblas/lib/libopenblas64_.0.dylib /var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/openblas/lib/libopenblas64_.a /var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/openblas/lib/libopenblas64_.dylib /var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/openblas/lib/libopenblas64_p-r0.3.23.dev.a /var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/openblas/lib/libopenblas64_p-r0.3.23.dev.dylib /var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/openblas/lib/pkgconfig /usr/local/lib
  + cp /etc/ssl/cert.pem /etc/ssl/certs /etc/ssl/cert.pem /etc/ssl/certs /var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/openblas/include/cblas.h /var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/openblas/include/f77blas.h /var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/openblas/include/lapack.h /var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/openblas/include/lapacke.h /var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/openblas/include/lapacke_config.h /var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/openblas/include/lapacke_mangling.h /var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/openblas/include/lapacke_utils.h /var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/openblas/include/openblas_config.h /usr/local/include
  cp: /etc/ssl/certs is a directory (not copied).
  cp: /etc/ssl/certs is a directory (not copied).
                                                              ✕ 4.31s
Error: Command bash ./tools/wheels/cibw_before_build.sh . failed with code 1. None
```

**Comment by henryiii** [comment]
It's failing in here:

https://github.com/numpy/numpy/blob/67539a40cb13bad56a650809bf10a49e905a250d/tools/wheels/cibw_before_build.sh#L29

It's setting basedir to:

```
basedir='/etc/ssl/cert.pem /etc/ssl/certs
  /etc/ssl/cert.pem /etc/ssl/certs
  /var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/openblas'
```

That means https://github.com/numpy/numpy/blob/main/tools/openblas_support.py is returning a bunch of values instead of one. Not all of the not-final print statements in that file use sys.stderr, which looks worrisome, but don't see one that's suspect. I suspect something else is printing those two lines.

**Comment by mayeut** [comment]
this is linked to the PyPy upgrade: https://foss.heptapod.net/pypy/pypy/-/issues/4010
cc @mattip

**Comment by mattip** [comment]
Sorry for the debug cruft on stdout when using `_ssl`. I hopefully will be releasing 7.3.14 soon with a fix to this.

**Comment by charris** [comment]
PyPy 7.3.14 is out.

**Comment by henryiii** [comment]
Update should be in #1712.

**Comment by joerick** [comment]
Just released v2.16.3 which contains the fixed pypy.
